packages/foundation/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@uifabric/foundation",
-  "version": "0.8.0",
+  "version": "0.109.0",
```

Comment:
Just curious, is this basically 100 release of "breathing room" for 6.0 branch publishes?

Reply:
Yes. We could choose a smaller number if you prefer.

Reply:
I don't think anyone expects there to be 100 releases out of the 6.0 branch, but it seemed better to give an excessive amount of space than a conservative amount then have to deal with it potentially running out.

Reply:
I was just guessing on the rationale here and wanted to confirm 😄